src/Symfony/Component/Console/Application.php

```diff
@@ -159,6 +159,8 @@ public function run(InputInterface $input = null, OutputInterface $output = null
      * @param OutputInterface $output An Output instance
      *
      * @return int 0 if everything went fine, or an error code
+     *
+     * @throws \Exception propagate the exception from `doRunCommand`
```

Comment:
this is current behaviour, just documenting it as `doRunCommand` is protected, so it's not part of public interface, which should be documented anyway...